Wikipedia/Code/TalkPageViewController+FindInPage.swift

```diff
@@ -0,0 +1,104 @@
+import Foundation
+
+extension TalkPageViewController {
+
+    // MARK: - Overrides
+
+    override var canBecomeFirstResponder: Bool {
+        return findInPageState.keyboardBar != nil
+    }
+
+    override var inputAccessoryView: UIView? {
+        return findInPageState.keyboardBar
+    }
```

Comment:
I have a few suggestions here to ensure the find in page bar plays nicely with the reply compose view.

Here I'm trying to keep the input accessory view from appearing while user is typing a reply:

```
    override var canBecomeFirstResponder: Bool {
        
        if replyComposeController.contentView == nil {
            return findInPageState.keyboardBar != nil
        }
        
        return false
    }
    override var inputAccessoryView: UIView? {
        if replyComposeController.contentView == nil {
            return findInPageState.keyboardBar
        }
        
        return nil
    }
```

Somewhat related - TalkPageViewController overrides a method to fix the content inset so that cells do not display beneath the reply compose view. But this is messing with the find in page content inset, so can you update that method to be:

```
override var keyboardIsIncludedInBottomContentInset: Bool {
    if replyComposeController.contentView != nil {
        return false
    } else {
        return true
    }
}
```

And feel free to refactor `replyComposeController.contentView != nil` into something like `replyComposeController.isShowing`, for code clarity.

Reply:
> I have a few suggestions here to ensure the find in page bar plays nicely with the reply compose view.

Unless I'm missing something, I think it may just be better to completely dismiss the find in page view in cases where the user interacts with view actions/controls that take over the page (like a reply, or pushing to a new view). I'd originally intended to do this in a follow-up PR, but I went ahead and added it here. Let me know what you think!

> TalkPageViewController overrides a method to fix the content inset so that cells do not display beneath the reply compose view.

Ahhh thank you, this was irritating me and I was incorrectly blaming `ViewController` for doing something wrong without digging into it. Updated!

Reply:
> Unless I'm missing something, I think it may just be better to completely dismiss the find in page view in cases where the user interacts with view actions/controls that take over the page.

Ah sorry, didn't catch that it would have been a part of that functionality. It's working well, thanks for doing it early!